collectors/charts.d.plugin/speedtest/README.md

```diff
@@ -103,3 +103,7 @@ should give you clues as to why the collector isn't working.
   ```bash
   ./charts.d.plugin speedtest
   ```
+  
+## Notes
+
+- Be very careful about setting low values of `update_every` when using this collector - by default it is `1800` which is one every 30 minutes. If you blindly set a very low value of `update_every` you will essentially be running continous speedtests that could severly impact your bandwidth and have related ingress/egress costs if using on a cloud VM for example. Read more [here](https://help.speedtest.net/hc/en-us/articles/360038679354-How-does-Speedtest-measure-my-network-speeds-) about how the test works.
```

Comment:
I think this belongs to the [first heading](https://github.com/netdata/community/tree/main/collectors/charts.d.plugin/speedtest#speedtest-monitoring-with-netdata). Who reads notes at the bottom of the page? 😄 

Also, can use Warning

> **Warning**: bla bla

Reply:
makes sense - done here: https://github.com/netdata/community/pull/68